persistence-cassandra-3.11/pom.xml

```diff
@@ -10,13 +10,13 @@
   <artifactId>persistence-cassandra-3.11</artifactId>
   <properties>
     <!-- If you update this, make sure to keep `cassandra.bundled-driver.version` in sync -->
-    <!-- 3.11.13 depends on 3.0.1: https://mvnrepository.com/artifact/org.apache.cassandra/cassandra-all/3.11.13 -->
-    <cassandra.version>3.11.13</cassandra.version>
+    <!-- 3.11.14 depends on 3.11.3: https://mvnrepository.com/artifact/org.apache.cassandra/cassandra-all/3.11.14 -->
+    <cassandra.version>3.11.14</cassandra.version>
     <!--
       The driver used internally by cassandra-all for UDFs (must match the version declared in
       cassandra-all's POM).
     -->
-    <cassandra.bundled-driver.version>3.0.1</cassandra.bundled-driver.version>
+    <cassandra.bundled-driver.version>3.11.3</cassandra.bundled-driver.version>
```

Comment:
As I stated in the other PR, this is wrong update:

![image](https://user-images.githubusercontent.com/10600041/211510353-aa8aa038-a9fb-4044-a240-db02d231e7cb.png)